### PR TITLE
Ticket 02

### PR DESCRIPTION
**Pull request by alaninfo23**
Stefany, please check if there is anything I can fix or improve?

### CODE REVIEW

File: src/components/Task.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const Task = ({ task, handleTaskClick, handleTaskDeletion }) => {
             style={ task.completed ? { borderLeft: '6px solid chartreuse' } : {}}
             data-testid={`TASK_CONTAINER_${task.title}`}
         >
-            <div className='task-title' onClick={() => handleTaskClick(task.id)}>
+            <div className='task-title' data-testeid='TASK-NAME' onClick={() => handleTaskClick(task.id)}>
```
Comment (norikoStefany-daitan) [inline src/components/Task.jsx:20]:
```suggestion
            <div className='task-title' data-testeid='TASK_NAME' onClick={() => handleTaskClick(task.id)}>
```

File: src/components/Task.jsx [hunk 2]
```diff
@@ -31,7 +31,9 @@ const Task = ({ task, handleTaskClick, handleTaskDeletion }) => {
                 </button>
 
                 <button 
-                  className='see-task-details-button' onClick={handleTaskDetailsClick}
+                  className='see-task-details-button' 
+                  data-testid='INFO_TASK'
```
Comment (norikoStefany-daitan) [inline src/components/Task.jsx:35]:
```suggestion
                  data-testid='INFO_TASK_BUTTON'
```

File: src/integration/tests/firstTest.test.jsx [hunk 3]
```diff
@@ -68,5 +66,29 @@ describe("Testes", () => {
     expect(screen.queryByTestId(`TASK_CONTAINER_${taskMsg}`)).not.toBeInTheDocument();
   });
 
+  it('validate if click button info it will show the back button and the title with task name.', () => {
```
Comment (norikoStefany-daitan) [inline src/integration/tests/firstTest.test.jsx:69]: Name suggestion for the scenario

```suggestion
  it('Patient should be able to open task details', () => {
```

File: src/integration/tests/firstTest.test.jsx [hunk 4]
```diff
@@ -68,5 +66,29 @@ describe("Testes", () => {
...
+     render(<App />);
+     const taskMsg = `testes de integração ${Date.now()}`;
+     const button_back = 'Voltar'
```
Comment (norikoStefany-daitan) [inline src/integration/tests/firstTest.test.jsx:72]:
```suggestion
     const backButton = 'Voltar';
```

File: src/integration/tests/firstTest.test.jsx [hunk 5]
```diff
@@ -68,5 +66,29 @@ describe("Testes", () => {
...
+
+     const input = screen.getByTestId('INPUT_TASK');
+     userEvent.clear(input);
+     userEvent.type(input, taskMsg);
+     const add_task = screen.getByTestId('ADD_BUTTON');
```
Comment (norikoStefany-daitan) [inline src/integration/tests/firstTest.test.jsx:77]:
```suggestion
     const addTask = screen.getByTestId('ADD_BUTTON');
```

File: src/integration/tests/firstTest.test.jsx [hunk 6]
```diff
@@ -68,5 +66,29 @@ describe("Testes", () => {
...
+     userEvent.click(add_task);
+     
+     //se colocar o ADD_BUTTON antes do within, o teste passa.
+     //expect(screen.getByTestId('ADD_BUTTON')).toHaveTextContent('Adicionar');
+     const task = within(screen.getByTestId(`TASK_CONTAINER_${taskMsg}`));
+     const info_task = task.getByTestId('INFO_TASK');
```
Comment (norikoStefany-daitan) [inline src/integration/tests/firstTest.test.jsx:83]:
```suggestion
     const infoTask = task.getByTestId('INFO_TASK');
```

File: src/integration/tests/firstTest.test.jsx [hunk 7]
```diff
@@ -68,5 +66,29 @@ describe("Testes", () => {
...
+     userEvent.click(info_task);
+     //expect(screen.queryByTestId(`TASK_CONTAINER_${taskMsg}`)).toBeInTheDocument();
+     //const backButton = task.getByTestId('BACK-BUTTON');
+     //expect(backButton).toHaveTextContent('Voltar');
```
Comment (norikoStefany-daitan) [inline src/integration/tests/firstTest.test.jsx:87]: I think we can remove these comments

File: src/integration/tests/firstTest.test.jsx [hunk 8]
```diff
@@ -49,7 +49,7 @@ describe("Testes", () => {
     expect(tasks).toHaveLength(2);
   });
 
-  it.skip('validar se quando deleto uma tarefa ela está sumindo da lista', () => {
+  it('validate if it is possible to delete a task.', () => {
```
Comment (norikoStefany-daitan) [inline src/integration/tests/firstTest.test.jsx:52]:
```suggestion
  it('User should be able to delete a task.', () => {
```

**Comment by alaninfo23** [comment]
Thanks Stefany, I already made the changes.

**Review by norikoStefany-daitan — COMMENTED** [review]
Just a few comments on the data-testid property which I didn't notice before

**Review by norikoStefany-daitan — APPROVED** [review]

**Comment by norikoStefany-daitan** [comment]
@alaninfo23 approved =)
Could you please update PR title and description? Then you can merge it. Thanks!

**Pull request merged**